src/SPIN/fix_nve_spin.cpp

```diff
@@ -197,18 +203,18 @@ void FixNVESpin::init()
   int count1 = 0;
   if (npairspin == 1) {
     count1 = 1;
-    spin_pairs[0] = (PairSpin *) force->pair_match("spin",0,0);
+    spin_pairs[0] = (PairSpin *) force->pair_match("^spin",0,0);
```

Comment:
@julient31 please note that not using the caret (^) can lead to false matches, same as for comparisons with `strstr()`. There may be styles that are not in the SPIN package and just by accident contain the string "spin" somewhere.
That is why we introduced the `utils::strmatch()` function, which has been applied here and is used internally by `Force::pair_match()` for inexact matches. Searching for styles that **start** with the string spin is much more specific.